Changelog.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased (in master)
 
+- **breaking** - Default for `line-padding` setting has been changed to `0`. If
+you've relied on it: set it to its' previous value `1` in your "init.moon".
```

Comment:
One final nitpick: Initialization can be written in either Lua or Moonscript, so instead of referring to 'init.moon' I suggest 'Howl configuration' or 'initialization configuration', etc.